appeals/web/src/server/appeals/appeal-details/final-comments/redact/redact.controller.js

```diff
@@ -0,0 +1,129 @@
+import logger from '#lib/logger.js';
+import { redactFinalCommentPage, confirmRedactFinalCommentPage } from './redact.mapper.js';
+import { patchFinalCommentRedaction } from './redact.service.js';
+import { patchFinalCommentsStatus } from '../view-and-review/view-and-review.service.js';
+import { COMMENT_STATUS } from '@pins/appeals/constants/common.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+const renderRedactFinalComment = (request, response) => {
+	const {
+		errors,
+		currentRepresentation,
+		currentAppeal,
+		session,
+		params: { finalCommentsType }
+	} = request;
+
+	if (!currentRepresentation || !currentAppeal) {
+		logger.warn('No representation or appeal found in session');
+		return response.status(404).render('app/404.njk');
+	}
+
+	const pageContent = redactFinalCommentPage(
+		currentAppeal,
+		currentRepresentation,
+		finalCommentsType,
+		session
+	);
+
+	return response.status(200).render('patterns/display-page.pattern.njk', {
+		errors,
+		pageContent
+	});
+};
+
+/** @type {import('@pins/express').RequestHandler<Response>}  */
+export const getRedactFinalComment = async (request, response) => {
```

Comment:
This is just an alias of the function above

Reply:
It's a good point. I write them like this out of habit now, as so many of these features need to share the rendering function between both GET and POST controllers, to handle any validation errors. That doesn't apply in this case though, so I've removed the redundant alias.